gallery/setupTests.js

```diff
@@ -12,7 +12,10 @@ jest.retryTimes(3);
 
 const toMatchImageSnapshot = configureToMatchImageSnapshot({
   customDiffDir: path.join(__dirname, 'test-results'),
-  customSnapshotIdentifier: ({ defaultIdentifier }) => defaultIdentifier
+  // eslint-disable-next-line no-undef
+  customSnapshotsDir: process.env.THEME === 'DARK' && path.join(__dirname, 'visualtests', '__image_snapshots_dark__'),
+  // eslint-disable-next-line no-undef
+  customSnapshotIdentifier: ({ defaultIdentifier }) => defaultIdentifier + (process.env.THEME === 'DARK' ? '-dark' : '')
```

Comment:
Once the lastest on the feature branch (which includes the latest from `main`) is merged into this branch, I think the eslint config will need `setupTests.js` added to the list of file patterns in the `overrides` config in the gallery's eslintrc

Reply:
> Once the lastest on the feature branch (which includes the latest from `main`) is merged into this branch, I think the eslint config will need `setupTests.js` added to the list of file patterns in the `overrides` config in the gallery's eslintrc

added to `overrides` here" [21ea274](https://github.com/sonatype/sonatype-react-shared-components/pull/808/commits/21ea2743176a4beb332f83f3c2b032a38d07021b)